infra/gcp/bash/ensure-releng.sh

```diff
@@ -44,6 +44,103 @@ if [ $# = 0 ]; then
     set -- "${PROJECTS[@]}"
 fi
 
+RELEASE_PROCESS_CLOUDBUILD_SVCACCT="648026197307@cloudbuild.gserviceaccount.com"
+STAGING_SIGNER_SVCACCT="krel-staging"
+K8s_ORG_SIGNER_SVCACCT="trust"
+PROMOTER_PROJECT="k8s-artifacts-prod"
+
+# This function ensures the cross-project impersonation
+# constraint is not enforced in the project. This is required
+# to allow principals from other projects to impersonate
+# service accounts from k8s-releng-prod 
+function ensure_cross_project_constraint() {
+    if [ $# != 1 ] || [ -z "$1" ]; then
+        echo "ensure_cross_project_constraint requires a project name" >&2
+        return 1
+    fi
+    local project="${1}"
+
+    # Read the project number
+    local project_id
+    project_id=$(gcloud projects describe "${project}" --format='value(projectNumber)')
+    
+    # Check if the constraint is enabled
+    local status_file
+    status_file=$(mktemp)
+    if gcloud --project="${project}" org-policies describe \
```

Comment:
This action requires we enable the `orgpolicy.googleapis.com` for this project. I don't think we want to do this now.

Reply:
If we don't switch this constraint off, the promoter and the release process cannot impersonate the signing service accounts. This means that they cannot generate the OIDC tokens they need and cannot sign any artifacts.